src/github.com/getlantern/flashlight/genconfig/genconfig.go

```diff
@@ -111,15 +111,15 @@ func main() {
 		}
 	}
 	if *proxiedSitesOutFile != "" {
-		proxiedSitesTmpl := loadTemplate(*proxiedSitesOutFile)
+		proxiedSitesTmpl := loadTemplate("proxiedsites.go.tmpl")
```

Comment:
It's quite interesting to me how `genconfig.go` could run correctly without using the correct template. However, the generated `proxiedsites.go` and `fallbacks.go` is used nowhere.